src/utils/conversion.ts

```diff
@@ -2244,3 +2249,65 @@ export function middlewarePortfolioToPortfolio(
   }
   return new NumberedPortfolio({ did, id: new BigNumber(kind) }, context);
 }
+
+/**
+ * @hidden
+ */
+export function fundraiserTierToTier(fundraiserTier: FundraiserTier): Tier {
+  const { total, price, remaining } = fundraiserTier;
+  return {
+    total: balanceToBigNumber(total),
+    price: balanceToBigNumber(price),
+    remaining: balanceToBigNumber(remaining),
+  };
+}
+
+/**
+ * @hidden
+ */
+export function meshFundraiserStatusToFundraiserStatus(
+  meshFundraiserStatus: MeshFundraiserStatus
+): FundraiserStatus {
+  if (meshFundraiserStatus.isLive) {
+    return FundraiserStatus.Live;
+  }
+
+  if (meshFundraiserStatus.isFrozen) {
+    return FundraiserStatus.Frozen;
+  }
+
+  return FundraiserStatus.Closed;
+}
+
+/**
+ * @hidden
+ */
+export function fundraiserToStoDetails(fundraiser: Fundraiser, context: Context): StoDetails {
+  const {
+    creator,
+    offering_portfolio: offeringPortfolio,
+    offering_asset: offeringAsset,
+    raising_portfolio: raisingPortfolio,
+    raising_asset: raisingAsset,
+    tiers,
+    venue_id: venueId,
+    start,
+    end,
+    status,
+    minimum_investment: minimumInvestment,
+  } = fundraiser;
+
+  return {
+    creator: new Identity({ did: identityIdToString(creator) }, context),
+    offeringPortfolio: meshPortfolioIdToPortfolio(offeringPortfolio, context),
+    offeringAsset: new SecurityToken({ ticker: tickerToString(offeringAsset) }, context),
+    raisingPortfolio: meshPortfolioIdToPortfolio(raisingPortfolio, context),
+    raisingAsset: new SecurityToken({ ticker: tickerToString(raisingAsset) }, context),
+    tiers: tiers.map(tier => fundraiserTierToTier(tier)),
+    venue: new Venue({ id: u64ToBigNumber(venueId) }, context),
+    start: momentToDate(start),
+    end: !end.isEmpty ? momentToDate(end.unwrap()) : undefined,
```

Comment:
You can use `isSome` instead of negating `isEmpty` (should be `isNone` instead of `isEmpty` in any case)